### DISCUSSION TITLE
Looking for suggestions on selecting software for downstream analysis.

### DISCUSSION THREAD

**Discussion by jeanchristophe13v** [vdemichev/DiaNN#1182] [Q&A] [answered]
I am super new in proteomics data analysis and I have seen several softwares for data analysis but really don't know which one to choose. (e.g. MaxQuant for upstream analysis, DIA-NN for whole analysis, various R packages for downstream analysis) 
I do know  _matrix_ is needed for visualization or further analysis, but as far as I am concerned, different softwares for raw data processing produce different matrix output, yes, conversion is also difficult for me.  I was wondering if it's okay to use DIA-NN for data preprocessing and produce the _matrix_ output for further analysis in R ? or use DIA-NN for the whole process procedure?

Could anyone please kindly offer some suggestions for this? Raw data processing or downstream analysis tutorials  also help!

**Comment by vdemichev** [comment, marked as answer]
Hi Jean,

We recommend DIA-NN to get from raw DIA data to a table of protein quantities in your samples. Based on DIA-NN output, you can do statistical analysis in R (best) or Python (also recommended) or even Excel. There are also online tools for statistics & plotting like Protifi Simplifi.

For DDA data, if you need to work with it, then we do like FragPipe. Also FragPipe might be good for DIA workflows with huge search spaces (~100M precursors), i.e. a spectral library is first created by FragPipe from DIA data and then can be used in DIA-NN.

Best,
Vadim

**Reply by jeanchristophe13v** [reply]
It's so kind of you! Have a nice day :)
